src/main/java/core/basesyntax/dao/FruitTransactionDaoImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.DataBase;
+
+public class FruitTransactionDaoImpl implements FruitTransactionDao{
+    @Override
+    public void add(String fruit, int quantity) {
+        DataBase.storage.put(fruit, quantity);
+    }
+
+    @Override
+    public void update(String fruit, int quantity) {
+        DataBase.storage.put(fruit, quantity);
+    }
+
+    @Override
+    public int get(String fruit) {
+        return DataBase.storage.get(fruit);
```

Comment:
Accessing the storage directly violates the encapsulation principle. Consider adding a method in the DataBase class to handle retrieval, which would also take care of the case when the fruit is not present in the storage (possibly returning 0 or throwing an exception).